### PR TITLE
fix(jaas) do not redirect to plan limit page on auth errors

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -90,9 +91,13 @@ export async function connect(id, password, roomName) {
     let { jwt } = state['features/base/jwt'];
     const { iAmRecorder, iAmSipGateway } = state['features/base/config'];
 
-    if (!iAmRecorder && !iAmSipGateway && !jwt && isVpaasMeeting(state)) {
-        jwt = await getJaasJWT(state);
-        APP.store.dispatch(setJWT(jwt));
+    if (!iAmRecorder && !iAmSipGateway && isVpaasMeeting(state)) {
+        await APP.store.dispatch(getCustomerDetails());
```
Comment (saghul) [inline connection.js:95]: Mobile doesn't use this file, is this something that we need to do on mobile too?

**Review by saghul — APPROVED** [review]
LGTM with a question.

File: connection.js (hunk 1 above)
Comment (quitrk) [reply connection.js:95]: I'm thinking yes, but it wasn't in the scope of this PR, I just prepared the groundwork

Comment (saghul) [reply connection.js:95]: Ack

**Pull request merged**